src/log.py

```diff
@@ -0,0 +1,107 @@
+"""
+log
+ 
+By: Norma Chang
+"""
+import json
+import datetime
+import time
+ 
+ 
+class Log:
+    """
+    Log class.
+ 
+    Generates a log file that contains the data from the properties of an instance of the leader/follower class.
+ 
+    Array of dictionaries is generated when the log file is loaded.
```

Comment:
This needs way more detail about what and where things are being stored

Reply:
removed